### DISCUSSION TITLE
Release 1.7.4

### DISCUSSION THREAD

**Discussion by fraxiinus** [fraxiinus/ReplayBook#310] [Releases]
## June Monthly Release

We have our metadata back! Now I feel really silly about the way I reacted when the new file format dropped. Riot Games exceeded my expectations and were able to get this data (which they don't even use) added back into replay files. Big thanks for Riot Flixxy and Riot ReinyRainy for getting this done for us. I appreciate the effect made to support third-party applications.

I will continue to work on ReplayBook for the time being. I have one or two more big features I want added still and now that ReplayBook works again, I have the motivation to keep working.

Thank you to everyone who use ReplayBook and supported me through the years.

### Installation Instructions

Install New: Simply extract the zip file and run ReplayBook.exe. A guided setup sequence will help you get started.
Update from Release X: Replace the previous ReplayBook.exe with the new version.

**Note:** It is recommended to clear your replay cache after updating.

![image](https://github.com/fraxiinus/ReplayBook/assets/8365233/2ac599cf-248c-4480-aae2-6050204c43f9)

### Patch Notes

**[New]** New Russian translation by @xnorxnorxnorxnor in https://github.com/fraxiinus/ReplayBook/pull/296
**[New]** New warning if Vanguard is running, replays cannot be played if Vanguard is active
**[New]** Added support for ROFL2 file type, introduced in patch 14.11, some new stats were added
**[Enhancement]** Updated French translation by @gzordrai in https://github.com/fraxiinus/ReplayBook/pull/297
**[Enhancement]** Updated Portuguese translation by @TroniPM in https://github.com/fraxiinus/ReplayBook/pull/300
**[Enhancement]** Update Simplified Chinese translation by @magcicada in https://github.com/fraxiinus/ReplayBook/pull/306

**Full Changelog**: https://github.com/fraxiinus/ReplayBook/compare/1.7.3...1.7.4

<hr /><em>This discussion was created from the release <a href='https://github.com/fraxiinus/ReplayBook/releases/tag/1.7.4'>Release 1.7.4</a>.</em>

**Comment by 1autodidact** [comment]
No effort is wasted, and hard work pays off; we can watch replays again. Thank you for your efforts.
